packages/atomic/src/components/insight/insight-icon-button/insight-icon-button.tsx

```diff
@@ -0,0 +1,26 @@
+import {FunctionalComponent, h} from '@stencil/core';
+import {Button, ButtonProps} from '../../common/button';
+interface Props extends Partial<ButtonProps> {
+  icon: string;
+  ariaLabel: string;
+  tooltip?: string;
+  onClick?: () => void;
+}
+
+export const InsightIconButton: FunctionalComponent<Props> = ({
```

Comment:
Nothing here looks overly specific to Insight. Perhaps an `icon` property could be added to `Button`, or `IconButton` could be added as a common component.

Reply:
cool, I'dd add it as a common component to make sure the styling is consistent!